test/cypress/integration/07-signin.test.js

```diff
@@ -8,7 +8,7 @@ describe('signin', () => {
     cy.visit('/signin?next=/testuseradmin');
     cy.get('input[name=email]').type('testuser+admin@opencollective.com');
     cy.get('button[type=submit]').click();
-    cy.getByDataCy('topbar-login-username').contains('Test User Admin', { timeout: 15000 });
+    cy.getByDataCy('collective-title').should('be.visible').contains('Test User Admin', { timeout: 15000 });
```

Comment:
I would add that to keep making sure user gets logged in:

```suggestion
    cy.getByDataCy('collective-title').should('be.visible').contains('Test User Admin', { timeout: 15000 });
    cy.assertLoggedIn();
```

Reply:
Done. 😄